ruma-events/src/room/message.rs

```diff
@@ -3,10 +3,15 @@
 use js_int::UInt;
 use ruma_common::StringEnum;
 use ruma_events_macros::MessageEventContent;
-use ruma_identifiers::EventId;
 use serde::{Deserialize, Serialize};
+use serde_json::Value as JsonValue;
 
-use super::{EncryptedFile, ImageInfo, ThumbnailInfo};
+use super::{
+    relationships::{
+        Annotation, InReplyTo, Reference, RelatesToJsonRepr, RelationJsonRepr, Replacement,
```

Comment:
I'd prefer `pub use`s for the types here that are part of the public interface of room messages, making the `relationships` module private.